framework/src/controller/ipc/ipc_socket.ts

```diff
@@ -12,56 +12,40 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-// eslint-disable-next-line
-/// <reference path="../../../external_types/pm2-axon/index.d.ts" />
-// eslint-disable-next-line
-/// <reference path="../../../external_types/pm2-axon-rpc/index.d.ts" />
-
-import * as axon from 'pm2-axon';
-import { PubSocket, PullSocket, PushSocket, RepSocket, SubSocket } from 'pm2-axon';
+import { Publisher, Subscriber, Router } from 'zeromq';
 import { join } from 'path';
-import { Server as RPCServer } from 'pm2-axon-rpc';
 
 export abstract class IPCSocket {
-	public pubSocket!: PushSocket | PubSocket;
-	public subSocket!: PullSocket | SubSocket;
-	public rpcServer: RPCServer;
+	public pubSocket!: Publisher;
+	public subSocket!: Subscriber;
 
 	protected readonly _eventPubSocketPath: string;
 	protected readonly _eventSubSocketPath: string;
-	protected readonly _actionRpcSeverSocketPath: string;
+	protected readonly _rpcSeverSocketPath: string;
+	private readonly _rpcServer: Router;
 
 	protected constructor(options: { socketsDir: string; name: string }) {
-		this._eventPubSocketPath = `unix://${join(options.socketsDir, 'pub_socket.sock')}`;
-		this._eventSubSocketPath = `unix://${join(options.socketsDir, 'sub_socket.sock')}`;
-		this._actionRpcSeverSocketPath = `unix://${join(
+		this._eventPubSocketPath = `ipc://${join(options.socketsDir, 'internal.pub.ipc')}`;
+		this._eventSubSocketPath = `ipc://${join(options.socketsDir, 'internal.sub.ipc')}`;
+		this._rpcSeverSocketPath = `ipc://${join(
 			options.socketsDir,
-			`${options.name}_rpc_socket.sock`,
+			`${options.name}.internal.rpc.ipc`,
```

Comment:
Can we extract this to a utility function, so we can have consistant socket names where ever needed.

Reply:
Will resolve it in https://github.com/liskhq/lisk-sdk/issues/6615